setenv.ts

```diff
@@ -0,0 +1,32 @@
+const { writeFile } = require('fs');
+const { argv } = require('yargs');
+
+// read environment variables from .env file
+require('dotenv').config({override: true});
+
+// read the command line arguments passed with yargs
+const environment = argv.environment;
+const isProduction = environment === 'prod';
+
+const targetPath = isProduction
+   ? `./src/environments/environment.prod.ts`
+   : `./src/environments/environment.ts`;
+
+// we have access to our environment variables
+// in the process.env object thanks to dotenv
+const environmentFileContent = `
+export const environment = {
+   production: ${isProduction},
+   REST_API_URL: "${process.env.REST_API_URL}",
+   SPRING_DATASOURCE_URL: "${process.env.SPRING_DATASOURCE_URL}",
```

Comment:
Maybe you should find a way to have some default values for these variables. In case `.env` file is not provided at all, or some values are missing, these values will be `undefined` or empty.
For example, the default `REST_API_URL=http://localhost:9966/petclinic/api/` is valid for local runs.